### PR TITLE
Bump self_cell version

### PR DESCRIPTION
**Pull request by Tristramg**
See https://github.com/Voultapher/self_cell/issues/49

(If you want, I can also bump the other crates, but as this is important, I’m making a separate PR)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "osmpbfreader"
-version = "0.16.0"
+version = "0.17.0"
```
Comment (TeXitoi) [inline Cargo.toml:3]: As self_cell is an implementation detail not exposed in public API, no need to do a semver break:

```suggestion
version = "0.16.1"
```

**Comment by TeXitoi** [comment]
v0.16.1 published

**Comment by TeXitoi** [comment]
Welcome. Are you using it now?

**Comment by Tristramg** [comment]
yes, i use it for osm4routing, that in turn is used for https://github.com/osrd-project/osrd

**Pull request merged**